packages/shared/src/api/dto/CreateTaskCommand.ts

```diff
@@ -0,0 +1,7 @@
+export interface CreateTaskCommand {
```

Comment:
remove frontend changes from this branch

Reply:
Done